src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -164,6 +165,29 @@ def _verify_openssl_version(lib):
         )
 
 
+def _verify_package_version(version):
+    # Occasionally we run into situations where the version of the Python
+    # package does not match the version of the shared object that is loaded.
+    # This may occur in environments where multiple versions of cryptography
+    # are installed and available in the python path. To avoid errors cropping
+    # up later this code checks that the currently imported package and the
+    # shared object that were loaded have the same version and raise an
+    # ImportError if they do not
+    so_package_version = Binding.ffi.string(
+        Binding.lib.CRYPTOGRAPHY_PACKAGE_VERSION
+    ).decode("ascii")
+    if version != so_package_version:
+        raise ImportError(
+            "The version of cryptography does not match with the loaded "
+            "shared object. This can happen if you have multiple copies of "
+            "cryptography installed in your Python path. Please try creating "
+            "a new virtual environment to resolve this issue. "
+            "Loaded python version: {0}, shared object version: {1}".format(
```

Comment:
Now that we've dropped py2.6 you can just use `{}` you don't need to number them